code/modules/mob/living/simple_animal/abnormality/he/fan.dm

```diff
@@ -52,7 +52,7 @@
 		if("Set it to 2")
 			observation_fail_message = observation_fail_message_3
 		else
-			observation_fail_message = initial(observation_success_message)
+			observation_success_message = initial(observation_success_message)
```

Comment:
is this right? Ya sure you dont want this instead:
```suggestion
			 observation_fail_message = initial(observation_fail_message)
```

Reply:
originally it was setting the fail message to the success message, the success message itself is never changed from what i can see in the code so i dont really see a reason in reseting it

Reply:
Tested locally, nope. Does not work. Eighter way.
`observation_success_message` and `observation_fail_message` are both purelly cosmetic and serve to tell the user some text, and are NOT checked for success or failure.
ya can check it at https://github.com/vlggms/lobotomy-corp13/blob/master/code/modules/mob/living/simple_animal/abnormality/_abnormality.dm#L561
Not a clue why it does not work and i blame observation code for this. Mark as resolved when ya read this, and please play-test your code.